chart/prometheus-msteams/templates/deployment.yaml

```diff
@@ -23,6 +23,10 @@ spec:
     {{ toYaml .Values.podAnnotations | indent 8 }}
     {{- end }}
     spec:
+      {{- if .Values.imagePullSecrets }}
+      imagePullSecrets:
+{{ toYaml .Values.imagePullSecrets | ident 8 }}
```

Comment:
This is not a bug. You will find both variants across different charts. I personally prefer the one which is not limiting your format to the way it is done "right now". In case `imagePullSecrets` ever changes it's interface (or adds features) you don't have to change the chart for it. As it allows it to be set independently and freely.

So are you sure you want me to change that?

Reply:
Oh I didn't know that, neither did I verified it I only verified against the kubernetes docs. Then it is fine for me. Still you have a typo: `ident` should be `indent`, see also the CI logs: https://github.com/prometheus-msteams/prometheus-msteams/pull/129/checks?check_run_id=1342268536.

If you fixed that, we can merge 👍

Reply:
hmm.. on my phone the CI was not showing the actual content, so I ignored it. Fixed. Happy merging :)

Reply:
Thanks for contributing @savar!